SDWebImage/Core/SDAnimatedImage.m

```diff
@@ -347,11 +347,21 @@ - (void)setSd_imageLoopCount:(NSUInteger)sd_imageLoopCount {
 }
 
 - (NSUInteger)sd_imageFrameCount {
-    return self.animatedImageFrameCount;
+    NSUInteger frameCount = self.animatedImageFrameCount;
```

Comment:
For non animated `SDAnimatedImage`, `self.animatedImageFrameCount` is nil, so the extra check is need.